src/Model.php

```diff
@@ -428,34 +417,36 @@ public function init(): void
         }
 
         // Declare our basic CRUD actions for the model.
-        $this->addAction('add', [
+        $this->addUserAction('add', [
             'fields' => true,
-            'modifier' => UserAction\Generic::MODIFIER_CREATE,
-            'scope' => UserAction\Generic::NO_RECORDS,
+            'modifier' => Model\UserAction::MODIFIER_CREATE,
+            'scope' => Model\UserAction::APPLIES_TO_NO_RECORDS,
```

Comment:
can we change the key to `applies_to` to?

Reply:
I would propose to do this in another PR as this gets into refactoring a core functionality and better be separate.

Reply:
I think it is ok to do it here if the change is small and if we are almost ready to merge this...